apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -57,7 +60,10 @@ export default class VirtualizedDetailView extends Component {
   };
 
   cellRenderer = ({columnIndex, key, rowIndex, style}) => {
-    const {section, scriptData} = this.props;
+    const {section, scriptData, studentLevelProgress} = this.props;
+    // Subtract 2 to account for the 2 header rows
+    // We don't want leave off the first 2 students.
+    const studentStartIndex = rowIndex-2;
```

Comment:
can we similarly have a const for stageId since we're calculating that index too?